README.md

```diff
@@ -13,12 +13,14 @@ $ (cd tracelib && npm install)
 $ (cd strategizer && npm install)
 $ (cd runtime && npm install)
 $ (cd strategy-explorer && bower install)
+$ (cd extension && npm install)
 
 ```
 
 ## Test
 ```
 $ (cd runtime && gulp)
+$ (cd extension && npm test)
```

Comment:
Would 'npm test' work for both runtime & the extension (or gulp work for both)?

The more consistent we can be the less surprising we'll be to new developers.

Reply:
It looks like 'npm test' will work for both, so will revise accordingly. I'm seeing a couple of tests fail right now at ToT but that seems to be the case with both 'gulp' and 'npm test'.